iib/web/healthcheck.py

```diff
@@ -0,0 +1,138 @@
+import time
+import logging
+import sqlalchemy
+import kombu
+
+from iib.exceptions import IIBError
+from iib.web import db
+from iib.workers.config import get_worker_config
+from iib.workers.tasks.celery import app
+
+
+log = logging.getLogger(__name__)
+
+
+def database_connection_check():
+    """
+    Check if the database connection is working.
+
+    :return: tuple (ok: bool, reason: str or None)
```

Comment:
could you add `:raises` and `rtype` to the doc string.

Reply:
Please refer other existing doc strings and keep the format uniform

Reply:
Sure! I can do that. :) 

There was a request to add these in another file, which I did. But I don't see this implemented in all files. That's why I didn't add it here; same thing with single & double quotes. I do understand that this is not a reason to not follow the format, but, if we want to enforce a rule, we should always apply it to all files, not just decide to enforce it when it suits us.

Reply:
I agree with you Zainab! It should be in all files. However I do not know how to automate these checks. So it might happen that developer forget to add it and it slips through code-review. 

If you see missing `:raises` and `rtype` feel free to create separate PR for that. Same with single/double quotes.

Or if you know a way how to automate this checks, please share it with us. It would be great to have these checks in our test suit - so it is really enforced.

Reply:
@lipoja you agreed and asked for it which means you believe it is important. but I don't see this. :) 

I reminded you about the code standard you have a concern about here and at the same time forgot about in one of your PRs. You changed it there and then sent another PR with the same missing code standard. 

I am not against it but let's all comply with the same standards. 

There is flake8 extension to lint for quotes (flake8-quotes) 

You could install it by pip and test it in your local. 

`pip install flake8-quotes
`

as an example, run this cmd: `flake8 --inline-quotes "'" iib/workers/tasks/utils.py`

the result should be similar to this: 

`iib/workers/tasks/utils.py:73:26: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:280:15: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:280:32: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:280:54: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:333:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:334:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:335:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:336:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:337:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:356:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:357:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:358:9: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:379:38: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:379:52: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:982:15: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:982:22: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:982:31: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1007:38: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:11: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:25: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:35: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:56: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:66: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1018:82: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1032:39: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1066:43: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1067:35: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1094:25: Q000 Double quotes found but single quotes preferred
iib/workers/tasks/utils.py:1094:54: Q000 Double quotes found but single quotes preferred`

there is also support for multiline. For ref: https://github.com/zheller/flake8-quotes

Reply:
@zanssa Thanks for pointing this out to us. I agree, it should be in all files. Maybe we missed it. Let's file an issue to add this enforcing? In the meanwhile, let's stay consistent going forward. I would really appreciate if you could make the change. Sorry to be nitpicking

Reply:
@zanssa Thanks for finding this out! Yes, as I said in https://github.com/release-engineering/iib/pull/316#discussion_r748579261 I do not like it and I prefer double quotes. Therefore I have hard time to write code with single quotes as well. It would be great if we can enforce it.